clippy_lints/src/lib.rs

```diff
@@ -352,6 +353,7 @@ pub fn register_pre_expansion_lints(store: &mut rustc_lint::LintStore) {
     store.register_pre_expansion_pass(|| box write::Write::default());
     store.register_pre_expansion_pass(|| box attrs::EarlyAttributes);
     store.register_pre_expansion_pass(|| box dbg_macro::DbgMacro);
+    store.register_pre_expansion_pass(|| box uninformative_asserts::UninformativeAsserts::default());
```

Comment:
Just chiming in to say this because @llogiq may not be aware of it, but we try to avoid pre-expansion lints lately. If possible, this should target the expanded version.

See [here](https://github.com/rust-lang/rust-clippy/pull/6167#discussion_r503846372) for some background.